view/GameView/ItemChoiceScreen.py

```diff
@@ -0,0 +1,237 @@
+                                                           "assets/GameView/GameViewTheme.json")
+
+        self.bottom_container = pygame_gui.core.UIContainer(
+            relative_rect=pygame.Rect((self.settings.window_width * .465, self.settings.window_height * .75),
+                                      (self.settings.window_width / 4, self.settings.window_height / 8)),
+            manager=self.manager)
+
+        self.char_img_container = pygame_gui.core.UIContainer(
+            relative_rect=pygame.Rect((self.settings.window_width * .35, self.settings.window_height * .15),
+                                      (self.settings.window_width * 3 / 2, self.settings.window_height / 4)),
+            manager=self.manager
+        )
+
+        self.char_name_container = pygame_gui.core.UIContainer(
+            relative_rect=pygame.Rect((self.settings.window_width * .35, self.settings.window_height * .15),
+                                      (self.settings.window_width * 3 / 2, self.settings.window_height / 4)),
+            manager=self.manager
+        )
+
+        self.gadget_img_container = pygame_gui.core.UIContainer(
+            relative_rect=pygame.Rect((self.settings.window_width * .35, self.settings.window_height * .45),
+                                      (self.settings.window_width * 3 / 2, self.settings.window_height / 4)),
+            manager=self.manager
+        )
+
+        self.gadget_name_container = pygame_gui.core.UIContainer(
+            relative_rect=pygame.Rect((self.settings.window_width * .35, self.settings.window_height * .45),
+                                      (self.settings.window_width * 3 / 2, self.settings.window_height / 4)),
+            manager=self.manager
+        )
+
+        self.__padding = self.bottom_container.rect.width / 10
+        self.__button_size = (self.bottom_container.rect.width / 3, self.bottom_container.rect.width / 12)
+        self.__img_size = (128, 128)
+        self.__img_pad = 2 * self.__img_size[0]
+
+        self.background = pygame.Surface((self.settings.window_width, self.settings.window_height))
+        self.background.fill(self.manager.ui_theme.get_colour(None, None, 'dark_bg'))
+
+        # font with which the descriptions are rendered
+        self.font = pygame.font.Font("assets/GameView/Montserrat-Regular.ttf", 20)
+
+        self._init_ui_elements()
+
+        logging.info("Itemchoice init done")
+
+    def draw(self) -> None:
+        self.manager.update(1 / self.settings.frame_rate)
+
+        self.window.blit(self.background, (0, 0))
+        self.manager.draw_ui(self.window)
+
+    def receive_event(self, event: pygame.event.Event) -> None:
+        self.manager.process_events(event)
+
+        if event.type == pygame.USEREVENT and event.user_type == pygame_gui.UI_BUTTON_PRESSED:
+            switcher = {
+                self.start_game_button: self.start_game_pressed,
```

Comment:
Button wird nicht gebraucht, oder?
Man muss ja x-mal ein Wahl treffen und in den EquipmentScreen wechselt man ja auf Nachricht des Servers.